src/pages/issues/index.tsx

```diff
@@ -47,6 +66,17 @@ const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
 };
 
 const Issues: FC = () => {
+    let splittedQueryParam = '';
+    const router = useRouter();
+    const dev = router?.query?.dev;
+    if (dev === 'true') {
+        const urlQueryParam = router?.query?.q as string;
+        if (urlQueryParam) {
+            const splittedQueryParamArr = urlQueryParam.split(':');
+            splittedQueryParam = splittedQueryParamArr[1];
```

Comment:
This will get in problem if/when there is more query added in future. Also will give unexpected result currently if we enter more queries under q currently. You can check how this is done in tasks route.

Reply:
According to the current implementation we are fetching issues on the basis of existence of only one parameter, which  is a search string, if more parameters are added in the future then it can be addressed better in a separate issue ticket.